lib/views/help/privacy.html.erb

```diff
@@ -2,573 +2,895 @@
-        openness, we will consider requests to remove the names of public
-        servants when it seems unlikely that the public interest will be harmed
-        by doing so.
+        The page does not display your email address, but does allow others to
+        get in touch with you via a messaging system.
       </p>
-      <p>This means that:</p>
-      <ul>
-        <li>
-          If you are a decision-maker of any seniority, or if you are responding
-          to an FOI request, we will not normally remove your details from
-          documents and emails sent by a public body. Accountability of
-          decision-making is at the heart of good government.
-        </li>
-        <li>
-          If you hold a junior, non-decision making post we will consider
-          requests to remove your details. Removing these details is time-
-          consuming for our volunteers, so please let us know why this really
-          matters to you. If we agree to remove your details we will take
-          reasonable steps to do so but in some cases may not be able to as it
-          can be difficult to remove material from some types of document, such
-          as images and PDFs.
-        </li>
-        <li>
-          We are happy try to assist FOI officers who want us to help them
-          remove their signatures from the website, but generally in such
-          circumstances we require replacement responses to be prepared and sent
-          to the requester before we will remove the original documents.
-          We are unable to edit attachments which have been released via our
-          service.
-        </li>
-      </ul>
       <p>
-        We are also of course subject to the law and obliged to
-        consider requests from anyone who considers our publication of their
-        personal information is, or is likely to, cause them unwarranted harm
-        or distress. (See <a href="http://www.legislation.gov.uk/ukpga/1998/29/section/10">
-        S10 of the Data Protection Act</a>.)
+        The page does not display details of alerts you have subscribed to,
+        although these are visible to you (and only you) when you are logged in.
       </p>
-    </dd>
-  </dl>
+    </li>
+  </ul>
+  <p>
+    This data (your name, email address, alert subscriptions) is not used for
+    any purpose other than the running of the site and will not be shared
+    externally.
+  </p>
+
+  <h2 id="retention_periods">
+    Retention periods
+    <a href="#retention_periods">#</a>
+  </h2>
+
+  <h3 id="retention_requests">
+    FOI requests
+    <a href="#retention_requests">#</a>
+  </h3>
+  <p>
+    As detailed above, sending an FOI request through this site results in your
+    name and email address being stored, along with the body of your request, on
+    our servers. This information is accessible only to site administrators. At
+    the current time, the policy is to retain this information indefinitely.
```

Comment:
This paragraph could be clearer. "This information is accessible only to site administrators" – actually the _name_ and _body of your request_ is public – its just the _email address_ that's admin only.